src/app/shared/libraries/accordion-item/accordion-item.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
more unit tests pleae. You can give it some mock template and run snapshot on it